skip-tests.json

```diff
@@ -700,6 +701,7 @@
     "cryptocom": {
         "skipWs": true,
         "skipMethods": {
+            "proxies": "they do not permit our proxy location",
```

Comment:
@ttodua cryptocom can't be accessed either in the UK or in Germany?

Reply:
I've removed them from this PR. now only KUNA skip involved in this PR